packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -0,0 +1,20 @@
+#!/opt/mesosphere/bin/python
+
+import os
+import subprocess
+import sys
+
```

Comment:
Can we add some comments here as to the purpose this file? Would be helpful for posterity.

Reply:
https://github.com/mesosphere/dcos/blob/urbanserj/dcos-net/packages/dcos-net/extra/dcos-net-setup.py#L3-L10